c9s.yaml

```diff
@@ -0,0 +1,169 @@
+# Manifest for everything specific to CentOS Stream CoreOS (SCOS)
+
+# ostree-layers:
+#   - overlay/05scos
+
+# CentOS Stream 9 repos + internal repos for now
+repos:
+  - baseos
+  - appstream
+  - rhel-8-server-ose
+  - walters-coreos-centos-stuff
+
+# We include hours/minutes to avoid version number reuse
+automatic-version-prefix: "412.91.<date:%Y%m%d%H%M>"
+# This ensures we're semver-compatible which OpenShift wants
+automatic-version-suffix: "-"
+# Keep this is sync with the version in postprocess
+mutate-os-release: "4.12"
+
+postprocess:
+  - |
+     #!/usr/bin/env bash
+     set -xeo pipefail
+
+     # Tweak /usr/lib/os-release
+     grep -v "OSTREE_VERSION" /etc/os-release > /usr/lib/os-release.rhel
+     OCP_RELEASE="4.12"
+     (
+     . /etc/os-release
+     cat > /usr/lib/os-release <<EOF
+     NAME="${NAME} CoreOS"
+     ID="scos"
+     ID_LIKE="rhel fedora"
+     VERSION="${OSTREE_VERSION}"
+     VERSION_ID="${OCP_RELEASE}"
```

Comment:
ref: https://github.com/openshift/machine-config-operator/pull/3179#discussion_r891157724

should we add `VARIANT_ID="coreos"` here?

Reply:
Will likely add it with my next rebase